content/docs/setup/kubernetes/helm-install/index.md

```diff
@@ -118,30 +124,30 @@ to manage the lifecycle of Istio.
 
 ## Uninstall
```

Comment:
Could you include the Uninstall sections for each option under the corresponding option section? That way, readers can find all the information pertinent to their option, including the uninstall, in a single place on the doc.

Reply:
Can I just add the link to corresponding option, since almost all doc share a common structure like this:
```
Prerequisites
...

Installation
...

Uninstall
...
```

Reply:
The problem is that if I am interested only in option 1, your approach would force me to scroll through all the steps of option 2 to get to the uninstall command for option 1. If instead you do:
- Prerequisites
- Option 1
    - Install
    - Uninstall
- Option 2
    - Install
    - Uninstall
Your readers can jump to the option relevant to them and don't need to scroll through the irrelevant content.